modules/claims_api/config/routes.rb

```diff
@@ -63,6 +63,9 @@
     mount OkComputer::Engine, at: '/healthcheck'
 
     post '/veteran-id:find', to: 'veteran_identifier#find', constraints: { find: /:find/ }
+    namespace :veterans do
+      get '/:veteran_id/claims', to: 'claims#index'
```

Comment:
same here, lets make this `:veteranId` instead

Reply:
Ah, good call.
Changed [here](https://github.com/department-of-veterans-affairs/vets-api/pull/6955/commits/e853ffbe0b7d9efdbebb9e96ca3ee927abe34c2b).